netbox/netbox/models/__init__.py

```diff
@@ -201,3 +205,6 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+    def get_absolute_url(self):
```

Comment:
Why does it need to be redefined here if it inherits from `NetBoxModel`?

Reply:
Because it doesn't inherit from NetBoxModel, it inherits from NetBoxFeatureSet, I don't see NetBoxModel in the chain unless I'm missing something?

Reply:
Oh, my mistake, I didn't expand upward enough and misread which class this was. Disregard.